### PR TITLE
Added AXIS Steel City economy with two modes

### PR DESCRIPTION
**Pull request by rhoun**
Axis Steel City cargo categories with two modes: 5 categories with raw food and materials together, and 6 categories where food and materials are separate. It now uses raw string to accommodate 6 categories, so the minimum OpenTTD version is 14. Other than that, the script logic is unchanged.

### CODE REVIEW

File: version.nut
Comment (F1rrel) [inline version.nut:1]: Please don't change the `version.nut`, I will do a release PR where these are changed based on the release type. This PR will require new major version due to the new parameter, it is not savegame compatible.

File: info.nut [hunk 1]
```diff
@@ -20,7 +20,7 @@ class MainClass extends GSInfo
     function GetURL()                   { return "https://www.tt-forums.net/viewtopic.php?f=65&t=87052"; }
     function GetVersion()               { return SELF_VERSION; }
     function GetDate()                  { return SELF_DATE; }
-    function GetAPIVersion()            { return "12"; }
+    function GetAPIVersion()            { return "14"; } //because of RAW_STRING
```
Comment (F1rrel) [inline info.nut:23]: Please remove the comment, it is enough to have it mentioned in the PR.

**Review by F1rrel — CHANGES_REQUESTED** [review]
Great changes. Finally they allowed the raw_string! I need to test it, but did not see any major issue with the code, only small things.

File: version.nut
Comment (rhoun) [reply version.nut:1]: File reverted. The intent was to save you from a few edits, but I forgot to consider that you likely have a workflow for versions and releases.

**Pull request merged**